### DISCUSSION TITLE
chess pieces to download from Github<title>

### DISCUSSION THREAD

**Discussion by cope61** [community/community#124164] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm looking for the folder that has images of chess pieces can anyone help me 
are inform where i needs to go to download them.

**Comment by CopperEagle** [comment]
Please check my answer on (your own) duplicate at #124163.

**Reply by CopperEagle** [reply]
@cope61 If it solved the problem, please consider marking it as the solution :smile:
